libr/cons/pal.c

```diff
@@ -132,7 +126,8 @@ R_API void r_cons_pal_init () {
 	cons->cpal.fname              = (RColor) RColor_RED;
 	cons->cpal.help               = (RColor) RColor_GREEN;
 	cons->cpal.input              = (RColor) RColor_WHITE;
-	cons->cpal.invalid            = (RColor) RColor_BRED;
+	cons->cpal.invalid            = (RColor) RColor_RED;
+	cons->cpal.invalid.attr       = 1u << 1;
```

Comment:
Why don't you keep BRED? Saves one line.

Reply:
macros are bad and should not pollute header files.

Reply:
Yes indeed, see my comment below then.